src/sdk-installer.ts

```diff
@@ -6,6 +6,8 @@ import * as fs from 'fs';
 const BUILD_TOOLS_VERSION = '30.0.0';
 const CMDLINE_TOOLS_URL_MAC = 'https://dl.google.com/android/repository/commandlinetools-mac-6609375_latest.zip';
 const CMDLINE_TOOLS_URL_LINUX = 'https://dl.google.com/android/repository/commandlinetools-linux-6609375_latest.zip';
+const BASE_ANDROID_SDK_URL_MAC = 'https://dl.google.com/android/repository/sdk-tools-darwin-4333796.zip';
+const BASE_ANDROID_SDK_URL_LINUX = 'https://dl.google.com/android/repository/sdk-tools-linux-4333796.zip';
```

Comment:
I haven't tried this but I'm guessing just installing the cmdline tools should be enough?